livesql/binlog.go

```diff
@@ -88,7 +88,7 @@ func NewBinlog(ldb *LiveDB, host string, port uint16, username, password, databa
 		return nil, err
 	}
 
-	syncer := replication.NewBinlogSyncer(&replication.BinlogSyncerConfig{
+	syncer := replication.NewBinlogSyncer(replication.BinlogSyncerConfig{
```

Comment:
why is this change necessary?

Reply:
I was just trying to fix the [build](https://travis-ci.org/samsarahq/thunder/builds/285862909?utm_source=github_status&utm_medium=notification) on my first commit. Seems like this API was changed [~22 days ago](https://github.com/siddontang/go-mysql/pull/163). Alternative would be to perhaps vendor this dependency